src/Compilers/CSharp/CscCore/project.lock.json

```diff
@@ -513,21 +513,7 @@
       },
       "System.Reflection.Metadata/1.2.0": {
         "dependencies": {
-          "System.Collections": "4.0.0",
```

Comment:
Agh. I have been very carefully attempting to not include all of the lock.json files.